### PR TITLE
Update its_45hd_heatpump.yaml

### PR DESCRIPTION
**Pull request by nicopret1**
Updated initial file with the changes listed in #866.  Also removed Linkage sensor id=106

### CODE REVIEW

**Comment by nicopret1** [comment]
Herewith the pull request for the following updates: 

1. Changed current temperature id from 26 to 21. The best temperature sensor to use for the current water temperature is the "Inlet temperature" (ID=21) due to the location of the sensor. ID=26 is the "Ambient temperature"
2. Added Suction temperature as secondary sensor (ID=20)
3. Added Ambient temperature as secondary sensor (ID=26)
4. Renamed "Exhaust temperature" to "Vent temperature" (ID=24)
5. Removed all electricity related sensors for Voltage, Current and Energy (i.e. ID's=18,101, 101, 104, 105,). The ITS heat pump currently do not have sensors for these aspects and all values are therefore=0.
6. Removed Chlorination (ID=30) since this type of heat pump is not a swimming pool heat pump. It is used as a water heater for the house (i.e geyser element alternative).
7. Removed Linkage sensor (ID=106)

**Comment by make-all** [comment]
Please add attributes to the climate entity for the sensors you are removing.  There may be other related models that do have those sensors, so users may still want to access the data.

**Comment by nicopret1** [comment]
Added attributes to the climate entity for the removed sensors in case other related models that do have those sensors.

File: custom_components/tuya_local/devices/its_45hd_heatpump.yaml [hunk 1]
```diff
@@ -58,20 +58,48 @@ primary_entity:
...
+      type: integer
+      name: Power
+      attribute: true
+    - id: 104
+      type: integer
+      name: Energy
+      attribute: true
+    - id: 105
+      type: boolean
+      name: Electricity monitor
```
Comment (make-all) [inline custom_components/tuya_local/devices/its_45hd_heatpump.yaml:97]: One further note: at attribute (dps) level, the name is not a UI exposed name (Home Assistant automatically capitalises and removes underscores - the opposite of what it does to entity names) so it should have underscores in place of spaces and by convention be all lower case.

Comment (make-all) [reply custom_components/tuya_local/devices/its_45hd_heatpump.yaml:97]: The lower case part of that also applies to the other attributes converted from sensors by the last commit.

Comment (nicopret1) [reply custom_components/tuya_local/devices/its_45hd_heatpump.yaml:97]: Thank you for your patience and educating me. Its still new to me. Will submit update shortly

Comment (nicopret1) [reply custom_components/tuya_local/devices/its_45hd_heatpump.yaml]: I noticed that I previously removed "Electricty today" sensor, but forgot to add the attribute to the climate entity. Will add it now and send upadte

**Pull request merged**